apps/src/code-studio/components/libraries/LibraryListItem.jsx

```diff
@@ -59,47 +96,66 @@ export default class LibraryListItem extends React.Component {
 
   render() {
     let library = this.props.library;
+
     return (
-      <div className="assetRow" style={styles.listItem}>
-        <span style={styles.libraryName}>{library.name}</span>
-        <span style={styles.description}>
-          {library.description}
-          <br />
+      <div style={styles.listItem}>
+        <div style={[{marginRight: 25}, styles.overflowEllipsis]}>
+          <Tooltip text={i18n.viewCode()} place="bottom">
+            <a onClick={this.viewCode} style={styles.libraryTitle}>
+              {library.name}
+            </a>
+          </Tooltip>
           {library.userName && (
-            <span style={styles.author}>Author: {library.userName}</span>
+            <div style={[styles.author, styles.overflowEllipsis]}>
+              <InlineMarkdown
+                markdown={i18n.authorName({name: library.userName})}
```

Comment:
`InlineMarkdown` is _not_ safe to use with externally-sourced strings.

We should probably either make it safe or rename it to make that clear

Reply:
Wait, nevermind! I misremembered; this actually is safe. Sorry for the false alarm!

Reply:
lol that's okay! i appreciate you double-checking 😄